lua/boo/config.lua

```diff
@@ -17,6 +17,7 @@ M.default = {
 	focus_on_open = true,
 	close_on_leave = true,
 	close_on_mouse_move = true,
+	open_on_mouse_hold = true,
```

Comment:
Maybe change the name to something like `open_on_cursor_hold`, since we are not triggering on mouse events.

Reply:
> Maybe change the name to something like `open_on_cursor_hold`, since we are not triggering on mouse events.

ah i forgot that